metrics/web/test/debug_menu/presentation/widgets/debug_menu_fps_monitor_toggle_test.dart

```diff
@@ -132,7 +133,7 @@ void main() {
 
         await tester.tap(toggleFinder);
 
-        verify(debugMenuNotifier.toggleFpsMonitor()).called(1);
+        verify(debugMenuNotifier.toggleFpsMonitor()).called(matchers.once);
```

Comment:
I believe it would be great to update the `throws an AssertionError if the given fps monitor view model is null` test to use the `takeException` method and the `isAssertionError` matcher to avoid using named imports and be more consistent in our tests.